app/controllers/membership_applications_controller.rb

```diff
@@ -264,5 +265,11 @@ def update_error(error_message)
 
   end
 
+  def send_new_membership_application_notice_to_admins(new_membership_application)
+    User.admins.each do |admin|
+      AdminMailer.member_application_received(new_membership_application, admin).deliver_later
```

Comment:
We're not using `.deliver_later` since the mailer is done via HTTP (REST) calls to the mailgun mailer.

Reply:
@weedySeaDragon I used `deliver_now`. If this is still wrong please let me know.